setup.py

```diff
@@ -9,7 +9,9 @@
     packages=find_packages(),
     install_requires=[
         'pyyaml>=4.2b1',
-        'smart-open==1.7.1'
+        'smart-open==1.8.0',
+        'click==7.0',
+        'dataclasses'
```

Comment:
Is `dataclasses` needed?

Reply:
It is below Python 3.7.

Reply:
I meant, are we actually data classes in our source?

Reply:
Yes, take a look at `utils.py`.

Reply:
Oh wow, the code there isn't used anymore :D haha.

We can do this in a separate PR if it's easier / cleaner, but the class at https://github.com/datamill-co/singer-runner/blob/master/singer_runner/utils.py#L17-L20
and functions in https://github.com/datamill-co/singer-runner/blob/master/singer_runner/utils.py#L25-L33 can be removed along with any references to them.

Reply:
Sounds good. I'll just make the change in this PR then.

Reply:
I've made the requested changes to allow for dataclasses to be removed as a dependency.